hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/TestReconUtils.java

```diff
@@ -166,9 +168,11 @@ public int read() throws IOException {
       }
     });
 
-    InputStream inputStream = new ReconUtils()
-        .makeHttpCall(httpClientMock, url);
-    String contents = IOUtils.toString(inputStream, Charset.defaultCharset());
+    String contents;
+    try (InputStream inputStream = new ReconUtils()
+        .makeHttpCall(httpClientMock, url)) {
```

Comment:
Can we modify the unit test to test the new makeHttpCall API?

Reply:
Done